.sbt-java-formatter.conf

```diff
@@ -0,0 +1,32 @@
+//ignored files
```

Comment:
add that they are autogenerated protobuf files in the comment

Reply:
Ok :)